README.md

```diff
@@ -140,6 +142,7 @@ The Production deploy will wait for the Tests and Staging deploy to finish befor
 This codebase is released under [the MIT License][mit].
 
 [mit]: LICENCE
+[dev-docs]:https://govwifi-dev-docs.cloudapps.digital
```

Comment:
Know if there is a plan to set up the DNS for this?

Reply:
In the future I think Paula said, but it can be changed if they see fit? @emileswarts